src/components/CustomButton.tsx

```diff
@@ -0,0 +1,8 @@
+import { styled } from "@mui/material/styles";
+import { Button } from "@mui/material";
+
+
+export const CustomButton = styled(Button)(({ theme }) => ({
+    color: theme.status.secondary,
+    borderColor: theme.status.secondary
+}));
```

Comment:
I haven't yet worked with styled components in this way but I do know it's a new good practice from MUI, this comment is for you to keep in mind that /components/ may not be the correct folder to hold these components when you arrive to a modern company.